### ISSUE TITLE
firejail crashes xreader in Mint 18.3

### ISSUE THREAD

**Issue by Prsman** [netblue30/firejail#2318]
This maybe a bug, not sure so bare with me.  Running Mint 18.3 Cinnamon, firejail 9.56LTS.
Xreader crashes. No pre-views, just Adobe icon.  In /etc/firejail/xreader.profile are the lines:

 # Breaks xreader on Mint 18.3
 # include /etc/firejail/whitelist-var-common.inc.  Un-commenting this line does not fix.

Is there a way to "disable" the xreader.profile to test if this is a bug?  Man pages are to complicated for me, trying not to crash my system.

**Comment by Prsman** [comment]
Closing this as it is not a bug.  Reverted to older version and problem still exists.

**Comment by Prsman** [comment]
I am re-opening this as removing firejail with apt-get purge fixes xreader.  Also text editor would crash.
I got an strace before removing firejail.  Xreader would crash with version 0.9.56-LTS and 0.9.54_1
[pdfreader.txt](https://github.com/netblue30/firejail/files/2713314/pdfreader.txt)

**Comment by Vincent43** [comment]
Side question: from where did you get LTS version?

**Comment by Vincent43** [comment]
Did you tried latest non-LTS version from https://sourceforge.net/projects/firejail/files/firejail/ ?

**Comment by Prsman** [comment]
Yes I tried the latest version.  Before install I removed all traces of firejail that I could find including the audio fix directory and file.  I installed and did not run fix sound and firecfg.  I did the fix sound manually.
Upon reboot I have all the symbolic links.  Firefox runs in firejail sandbox.  Xreader crashes, ( does not run).  Will try firejail --clean and test.  FYI, I run firefox from the panel.  After deleting firejail, the firefox icon on the panel was removed.  I had modded the launcher to firejail firefox.  After install and reboot the icon on the panel is back.  I did ls -al on /usr/local/bin and the date is not today's date. So I did not remove everything.  Will try the clean command and report back.  Thanks
